third_party/amd/lib/TritonAMDGPUToLLVM/Utility.h

```diff
@@ -30,13 +32,19 @@ Value llGetPid(Location loc, RewriterBase &rewriter, ModuleOp moduleOp,
 // Loads from shared or global memory with predication.
 // `otherElems` is used to mask out the elements that are not loaded
 Value llLoad(RewriterBase &rewriter, Location loc, Value ptr, Type elemTy,
-             Value pred, Value falseVal, int64_t alignmentBytes = 0,
-             triton::CacheModifier cm = triton::CacheModifier::NONE);
+             Value pred, Value falseVal,
+             std::optional<triton::AMD::TargetInfo> targetInfo = std::nullopt,
```

Comment:
We can always require `targetInfo` to be existing? Any reason not doing that?

Reply:
I didn't want to change the calls in `AMDTargetInfo`, but it was mostly laziness from my part.